pallets/asset/src/benchmarking.rs

```diff
@@ -289,153 +289,122 @@ benchmarks! {
+        let (owner, ticker) = owned_ticker::<T>();
+        let docs = iter::repeat(make_document()).take(MAX_DOCS_PER_ASSET as usize).collect::<Vec<_>>();
+        Module::<T>::add_documents(owner.origin().into(), docs.clone(), ticker)
             .expect("Documents cannot be added");
 
-        let remove_doc_ids = (1..d).map(|i| DocumentId(i-1)).collect::<Vec<_>>();
-
-    }: _(owner.origin, remove_doc_ids.clone(), ticker.clone())
+        let remove_doc_ids = (1..d).map(|i| DocumentId(i - 1)).collect::<Vec<_>>();
+    }: _(owner.origin, remove_doc_ids.clone(), ticker)
     verify {
         for i in 1..d {
-            assert_eq!(<AssetDocuments>::contains_key( &ticker, DocumentId(i-1)), false);
+            assert_eq!(AssetDocuments::contains_key( &ticker, DocumentId(i-1)), false);
         }
     }
 
-     set_funding_round {
+    set_funding_round {
         let f in 1 .. T::FundingRoundNameMaxLength::get() as u32;
 
-        let owner = UserBuilder::default().generate_did().build("owner");
-        let ticker = make_asset::<T>(&owner);
-
+        let (owner, ticker) = owned_ticker::<T>();
         let fundr = FundingRoundName::from(vec![b'X'; f as usize].as_slice());
-     }: _(owner.origin, ticker.clone(), fundr.clone())
+     }: _(owner.origin, ticker, fundr.clone())
      verify {
-        assert_eq!( Module::<T>::funding_round(ticker), fundr);
+        assert_eq!(Module::<T>::funding_round(ticker), fundr);
      }
 
-
      update_identifiers {
-         let i in 1 .. MAX_IDENTIFIERS_PER_ASSET;
-
-         let owner = UserBuilder::default().generate_did().build("owner");
-         let ticker = make_asset::<T>(&owner);
+        let i in 1 .. MAX_IDENTIFIERS_PER_ASSET;
 
-         let identifiers: Vec<AssetIdentifier> =
-             iter::repeat(AssetIdentifier::cusip(*b"037833100").unwrap()).take(i as usize).collect();
+        let (owner, ticker) = owned_ticker::<T>();
 
-     }: _(owner.origin(), ticker.clone(), identifiers.clone())
+        let identifiers: Vec<_> = iter::repeat(AssetIdentifier::cusip(*b"037833100").unwrap())
+            .take(i as usize)
+            .collect();
+        let identifiers2 = identifiers.clone();
+     }: _(owner.origin(), ticker, identifiers)
      verify {
-        assert_eq!( Module::<T>::identifiers(ticker), identifiers);
+        assert_eq!(Module::<T>::identifiers(ticker), identifiers2);
      }
 
      add_extension {
-         let owner = UserBuilder::default().generate_did().build("owner");
-         let ticker = make_asset::<T>(&owner);
-         let ext_details = make_extension::<T>(false);
-         let ext_id = ext_details.extension_id.clone();
-     }: _(owner.origin(), ticker.clone(), ext_details.clone())
+        let (owner, ticker) = owned_ticker::<T>();
+        let ext_details = make_extension::<T>(false);
+        let ext_id = ext_details.extension_id.clone();
+     }: _(owner.origin(), ticker, ext_details.clone())
```

Comment:
Perhaps the call to `owner.origin()` and `ext_details.clone()` could be moved to the setup block?